lib/meson.build

```
@@ -17,7 +17,7 @@ libswitchboard_deps = [
     gtk_dep,
 ]
 
-libswitchboard_lib = shared_library('switchboard-2.0',
+libswitchboard_lib = shared_library('switchboard-3.0',
```

```suggestion
libswitchboard_lib = library('switchboard-3',
```
Just following what gtk has done here